packages/flutter/lib/src/rendering/child_view.dart

```
@@ -20,7 +20,7 @@ import 'object.dart';
 mojom.ViewProxy _initViewProxy() {
   int viewHandle = ui.takeViewHandle();
   assert(() {
-    if (viewHandle == 0)
+    if (viewHandle == core.MojoHandle.INVALID)
       debugPrint('Child view are supported only when running in Mojo shell.');
```

this should probably throw an exception, no?